api/v1beta1/network_disruption.go

```diff
@@ -150,23 +172,118 @@ type NetworkDisruptionCloudServiceSpec struct {
 	ConnState string `json:"connState,omitempty"`
 }
 
-// Validate validates args for the given http filters.
-func (s *NetworkHTTPFilters) Validate() error {
-	if s.Path != "" {
-		if len(s.Path) > MaxNetworkPathCharacters {
-			return fmt.Errorf("the path specification at the network disruption level is not valid; should not exceed 100 characters")
+func (p HTTPPath) validate() error {
+	if len(p) > MaxNetworkPathCharacters {
+		return fmt.Errorf("the paths specification at the network disruption level is not valid; should not exceed 100 characters")
+	}
+
+	if regexp.MustCompile(`\s`).MatchString(string(p)) {
+		return fmt.Errorf("the paths specification at the network disruption level is not valid; should not contains spaces")
+	}
+
+	if string(p[0]) != DefaultHTTPPathFilter {
+		return fmt.Errorf("the paths specification at the network disruption level is not valid; should start with a /")
+	}
+
+	return nil
+}
+
+func (paths HTTPPaths) isNotDefault() bool {
+	if len(paths) == 0 || len(paths) > 1 {
+		return false
+	}
+
+	return paths[0] != DefaultHTTPPathFilter
+}
+
+func (h HTTPMethods) isNotEmpty() bool {
+	return len(h) >= 1
+}
+
+// validate validates args for the given http filters.
+func (s *NetworkHTTPFilters) validate() (retErr error) {
+	if s.DeprecatedPath != "" {
+		retErr = multierror.Append(retErr, fmt.Errorf("the Path specification at the HTTP network disruption level is deprecated; use Paths HTTP field instead"))
```

Comment:
```suggestion
		retErr = multierror.Append(retErr, fmt.Errorf("the Path specification at the HTTP network disruption level is no longer supported; use Paths HTTP field instead"))
```

it's fine to label these as deprecated internally, but technically deprecated means we are still supporting something "for now" but will remove it later. However, these are already removed, so we should just be clear

Reply:
Completely agree with you! It's a mistake from my part...